src/jvm/test/resources/expected_logs/array_rw_run_with_lambda.txt

```diff
@@ -0,0 +1,37 @@
+= Invalid execution results =
+| -------------------------------------- |
+|                Thread 1                |
+| -------------------------------------- |
+| run(() -> R): IllegalStateException #1 |
+| -------------------------------------- |
+
+---
+The number next to an exception name helps you find its stack trace provided after the interleaving section
+---
+
+The following interleaving leads to the error:
+| -------------------------------------- |
+|                Thread 1                |
+| -------------------------------------- |
+| run(() -> R): IllegalStateException #1 |
```

Comment:
Can we not show `run(..)` here?

Reply:
I've implemented it, not without some crutches. 
We should really refactor `TraceReporter.kt` at some point, it becomes really painful to maintain and modify it...